### PR TITLE
Makes toy reality pierces generate names randomly.

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
So if you have tooltips turned on you can just draw cursor on it and you already know that its fake. Now you wont know because it'll have random generated name based on reality pierces string.
Examples of how it was and how it will look like:
image
image
## Why It's Good For The Game
You can fool someone with this. Should be fun.
## Changelog
:cl:
add: Added random generated names of toy reality pierces, so you can fake heretics and fool somebody.
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1310,6 +1310,10 @@
 	icon_state = "pierced_illusion"
 	item_flags = NO_PIXEL_RANDOM_DROP
 
+/obj/item/toy/reality_pierce/Initialize(mapload)
+	. = ..()
+	name = "\improper" + pick(strings(HERETIC_INFLUENCE_FILE, "drained")) + " " + pick(strings(HERETIC_INFLUENCE_FILE, "prefix")) + " " + pick(strings(HERETIC_INFLUENCE_FILE, "postfix"))
```
Comment (Mothblocks) [inline code/game/objects/items/toys.dm:1315]: Make this a proc that influences use too.

**Comment by private-tristan** [comment]
Will there be a way to check whether pierced realities are real or not now? The only ways I can see (that don't run the risk of brain damage or missing limbs) is pushbroom, bodybags, and conveyors.

**Comment by Mothblocks** [comment]
if you examine a real reality you get brain damage

**Comment by Helg2** [comment]
> if you examine a real reality you get brain damage @Helg2

Even better!

**Comment by Helg2** [comment]
> Will there be a way to check whether pierced realities are real or not now? The only ways I can see (that don't run the risk of brain damage or missing limbs) is pushbroom, bodybags, and conveyors.

Oh, and as appears there won't be such way. Suffer

**Comment by Mothblocks** [comment]
i guess i do think its a bit weird that this toy is much much much harder (read: more dangerous) to verify the legitimacy of...every other toy can be validated by any person

**Comment by Helg2** [comment]
> i guess i do think its a bit weird that this toy is much much much harder (read: more dangerous) to verify the legitimacy of...every other toy can be validated by any person

Well, i mean yeah, but without this change its just useless at all.

**Comment by Mothblocks** [comment]
Yeah it's a toy lol

**Comment by Helg2** [comment]
> Will there be a way to check whether pierced realities are real or not now? The only ways I can see (that don't run the risk of brain damage or missing limbs) is pushbroom, bodybags, and conveyors.

Oh, i got an idea how to find it out, ask ai as it cant see it or check it on cams console if it works this way.

**Comment by Mothblocks** [comment]
You do not have to do that for any other toy, I'm getting more and more convinced this isn't a good idea without a tell as intuitive as every other toy in the game

**Comment by Jackal-boop** [comment]
You can pull the toys but not the real realities. So I think it's fine

**Comment by Mothblocks** [comment]
Touching realities chops off your arm, feels fairly silly to not expect pulling to do the same

**Comment by ArthuroPep** [comment]
The brain damage of examining a pierced reality is minimal, i think it's fine

**Comment by Helg2** [comment]
> You do not have to do that for any other toy, I'm getting more and more convinced this isn't a good idea without a tell as intuitive as every other toy in the game

I fooled captain today with toy emag, just sharing :D. I mean, examining is still works as clock, the risk is kinda low as its just 10 brain damage which will hardly occure in anything as mild traumas start from 10 brain damage. And a mood debuff lasts only for 4 minutes so risks aren't that high.

**Comment by Mothblocks** [comment]
You haven't really convinced me that this is a good idea and this is a pretty big divergence from our current toy balance

**Comment by Helg2** [comment]
The main question is why not?

**Comment by Helg2** [comment]
> You haven't really convinced me that this is a good idea and this is a pretty big divergence from our current toy balance

Will argumentation that this will enforce people to use examine on it and not just right-click it for it's name? This will also make this toy at least worth of it's decreased drop chance in arcades.

**Comment by Mothblocks** [comment]
I don't want people to have to go out of their way to figure out if it's real or not because that's unlike any other toy we have right now. Examining is considered going out of your way because examining the real ones gives you brain damage.

**Comment by Helg2** [comment]
> I don't want people to have to go out of their way to figure out if it's real or not because that's unlike any other toy we have right now. Examining is considered going out of your way because examining the real ones gives you brain damage.

How do you think idea with reality smashes in heretic_boxes will pass? I think it would look cool
image
image

**Comment by Mothblocks** [comment]
I am not sure I understand

**Comment by Helg2** [comment]
> I am not sure I understand

There will be like list from reality smash and reality pierce toys to pick for heretic_box, and reality smash will just look like undiscovered heretic influence with similar name.

**Comment by Mothblocks** [comment]
But that doesn't seem to address my concerns

**Pull request closed without merging**